### PR TITLE
fix: Default config values

### PR DESCRIPTION
**Pull request by makingthematrix**
A few buildConfigFields and manifestPlaceholders  were missing from app/build.gradle. This usually is not a problem, but after removing Gradle's cache and doing `./gradlew clean`, I wasn't able to compile the app again without them.

#### APK
[Download build #1489](http://10.10.124.11:8080/job/Pull%20Request%20Builder/1489/artifact/build/artifact/wire-dev-PR2665-1489.apk)
[Download build #1569](http://10.10.124.11:8080/job/Pull%20Request%20Builder/1569/artifact/build/artifact/wire-dev-PR2665-1569.apk)

### CODE REVIEW

**Comment by makingthematrix** [comment]
build again

File: app/build.gradle [hunk 1]
```diff
@@ -96,7 +96,13 @@ android {
         applicationId "com.waz.zclient"
         testInstrumentationRunner "com.waz.background.TestRunner"
 
-        manifestPlaceholders = [customURLScheme: buildtimeConfiguration.configuration.custom_url_scheme]
+        manifestPlaceholders = [
+            customURLScheme   : buildtimeConfiguration.configuration.custom_url_scheme,
+            applicationLabel  : "Wire Dev",
+            applicationIcon   : "@mipmap/ic_launcher_wire_dev",
+            sharedUserId      : "",
+            internal_features : "true"
```
Comment (android10) [inline app/build.gradle]: What is this flag doing? What is the difference between ```internal_features``` and ```developer_features_enabled```?

In any case I would be a bit defensive here and put this to ```false``` by default. Does that make sense? Or at least write a test checking this out in order to not accidentally ship any WIP or internal funcionality...

File: default.json [hunk 2]
```diff
@@ -37,5 +37,8 @@
   "block_on_password_policy": false,
   "wipe_on_cookie_invalid": false,
   "force_private_keyboard": false,
-  "password_max_attempts": 0
+  "password_max_attempts": 0,
+  "developer_features_enabled": false,
```
Comment (android10) [inline default.json]: This is what I mean with the above comment.

File: app/build.gradle (hunk 1 above)
Comment (makingthematrix) [reply app/build.gradle]: Good catch. I don't think it's used at all in practice. There is a line in `AndroidManifest.xml`:
```
        <!-- Internal features from SE -->
        <meta-data android:name="INTERNAL" android:value="${internal_features}"/>
```
but then we don't use that `INTERNAL` flag anywhere. I guess I can delete it.

Comment (android10) [reply app/build.gradle]: Go for it if you feel comfortable/safe. Is this something that maybe we are using for custom builds? It does not look like it doh....Just thinking loudly. ;)

**Comment by makingthematrix** [comment]
build again

File: app/build.gradle (hunk 1 above)
Comment (makingthematrix) [reply app/build.gradle]: done

File: default.json (hunk 2 above)
Comment (makingthematrix) [reply default.json]: ok

**Comment by android10** [comment]
@makingthematrix is this something we want to merge? It looks like it is 25 days old.

**Pull request merged**